src/server/server.ts

```diff
@@ -184,6 +189,25 @@ async function serveArticlePost(
 	}
 }
 
+async function serveEditionsArticlePost(
+	req: Request,
+	res: ExpressResponse,
+	next: NextFunction,
+): Promise<void> {
+	try {
+		// The "req.body" should contain a 'Content' object which fetched by the
+		// Edition backend from the capi
+		const content = await capiContentDecoder(req.body);
+		const mockedRenderingRequest: RenderingRequest = {
```

Comment:
I think you could probably just call this `renderingRequest`? "mocked" may cause confusion because it's typically only something that's done in tests?